infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftEnvironmentProvisioner.java

```diff
@@ -86,7 +88,8 @@ public OpenShiftEnvironmentProvisioner(
       ProxySettingsProvisioner proxySettingsProvisioner,
       ServiceAccountProvisioner serviceAccountProvisioner,
       CertificateProvisioner certificateProvisioner,
-      VcsSshKeysProvisioner vcsSshKeysProvisioner) {
+      VcsSshKeysProvisioner vcsSshKeysProvisioner,
+      GitUserProfileProvisioner gitUserProfileProvisioner) {
```

Comment:
These changes seem unrelated to the bug this PR is fixing. They seem add support for the git prefs provisioning to Openshift which IMHO is a whole another issue.

Reply:
Good, I'll provide dedicate PR for this registration. @amisevsk fyi.

Reply:
@metlos @amisevsk https://github.com/eclipse/che/pull/14488

Reply:
:+1: